Storage/Processor/Processor.php

```diff
@@ -216,6 +216,11 @@ public function getResource(Config $config, string $filename)
                 return fopen($config->getFilename(), 'r');
             }
 
+            //it means that we wont be able to locate any file specified in the filenames config
```

Comment:
dead code should be removed. 

We could encapsulate the if statement in a function `$this->definedInConfig($config);` and then the comment is no longer necessary